ReusableViews/Classes/ReusableView.swift

```diff
@@ -61,7 +61,7 @@ public extension NibLoadableView where Self: UIView {
   /// - Returns: an instantiated view
   static func create(viewIndex: Int = 0, owner: Any? = nil, options: [AnyHashable: Any]? = nil) -> Self {
     let bundle = Bundle(for: Self.self)
-    guard let view = bundle.loadNibNamed(Self.nibName, owner: owner, options: options)?[viewIndex] as? Self else {
+    guard let view = bundle.loadNibNamed(Self.nibName, owner: owner, options: options as? [UINib.OptionsKey : Any])?[viewIndex] as? Self else {
```

Comment:
Can we split this line into multiple statements? There's multiple casts and unsafe method chaining - it's getting confusing.

Reply:
Done!